sql-statements/sql-statement-split-region.md

```diff
@@ -323,17 +333,17 @@ Splitting Regions for partitioned tables is the same as splitting Regions for or
     +-----------+---------------+---------------+-----------+-----------------+------------------+------------+---------------+------------+----------------------+------------------+
     ```
 
-4. You can also split Regions for the index of each partition. For example, you can split the `[1000,10000]` range of the `idx` index into two Regions:
+4. You can also split Regions for the index of each partition. For example, you can split the `[1000,10000]` range of the `idx` index into two Regions. The example statement is as follows:
 
     {{< copyable "sql" >}}
 
     ```sql
     split partition table t index idx between (1000) and (10000) regions 2;
     ```
 
-#### Split Region for a single partition
+#### Examples of Split Region for a single partition
 
-You can specify the partition to be split. See the following usage example:
+You can specify the partition to be split. If you want to create a Range table divided into three partitions, the example statement is as follows:
```

Comment:
This sentence should be in **Line 348**: Suppose that you want to create a Range partitioned table split into three partitions. The example statement is as follows: